src/shared/bg_pmove.cpp

```diff
@@ -1737,21 +1730,23 @@ static void PM_WaterMove()
 #endif
 	PM_Friction();
 
-	scale = PM_CmdScale( &pm->cmd, true );
+	float scale = PM_CmdScale( &pm->cmd, true );
 
 	//
 	// user intentions
 	//
 
-	for ( i = 0; i < 3; i++ )
+	vec3_t wishvel;
```

Comment:
I think you could get rid of one of wishvel or wishdir in all these functions

Reply:
well, that'll be for another day